generation_aggregation.md

```diff
@@ -0,0 +1,57 @@
+# Aggregate Metering Generation Uploads
+
+This help article will walk through how to upload generation for aggregate metered facilities with a fuel split via the system interface.
+
+## What is an Aggregate Meter Generator Registration?
+In the case where there is more than one Generating Unit associated with a single meter, the Generator Owner or Responsible Party can register more than one Generating Unit under one registration as a multi-fuel registration if, and only if, all the Generating Units have the same essential generation characteristics. If the Generating Units associated with a single meter do not have the exact same essential generation characteristics, the Generator Owner or Responsible Party has two choices: they can either install a new meter(s) such that all Generating Units associated with a single meter have the same generation characteristics, or they can use the Aggregate Metering feature in the Generator Registration to create an aggregate metering group to designate a percentage allocation on an MWh output basis for each different Generating Unit(s) associated with the meter. 
+
+Generators part of an Aggregate Metering Group will have generation data reported based on the Aggregate Meter ID shared by all generators in the group. The User will then issue certificates using the default meter split based on the Pro Rata Share of the registered Nameplate Capacities from each generator registration in the group or using the meter split entered by the User.  
+
+Generation for these facilities can now be uploaded via a CSV or through the system user interface. 
+
+## Can Distributed Generation projects aggregate their registrations together? 
+Distributed Generation projects that would like to aggregate their generators should not use the Aggregate Meter Registration. M-RETS has developed Distributed Generation Groups to register multiple DG projects up to 1 MW. If you do not see the 'Distributed Generation Group' option in the vertical menu, contact the M-RETS Administrator systemadmin@mrets.org 
```

Comment:
It would be awesome if we could add a link to DGGs once we have a help article for them